src/Akeneo/Pim/Structure/Bundle/Resources/workspaces/settings-ui/src/components/datagrids/attribute-groups/ModalMassDeleteAttributeGroups.tsx

```diff
@@ -45,16 +63,41 @@ const ModalMassDeleteAttributeGroups = ({attributeGroups, onConfirm}: MassDelete
         >
           <ModalContent>
             <p>
-              {translate('pim_enrich.entity.attribute_group.mass_delete.confirm', {
-                count: String(attributeGroups.length),
-              })}
+              {translate(
+                'pim_enrich.entity.attribute_group.mass_delete.confirm',
+                {
+                  count: String(selectedAttributeGroups.length),
```

Comment:
We didn't need to cast it to string
```suggestion
                  count: selectedAttributeGroups.length,
```